indico/htdocs/sass/modules/_reviewing.scss

```diff
@@ -331,7 +331,7 @@
 
             .details-content-row {
                 color: $dark-gray;
-                padding: 5px 10px 5px calc(20px + 1em);
+                padding: 5px 10px;
```

Comment:
After this change the UI looks like this:
![image](https://cloud.githubusercontent.com/assets/1579899/26551500/2e4a5108-4483-11e7-890c-afedf03d094f.png)

We want it to look like this: 
![image](https://cloud.githubusercontent.com/assets/1579899/26551652/d5626232-4483-11e7-829d-0b9d37c4cae7.png)

This can be achived with:
``` sass
padding: 5px 10px 5px calc(22px + 2em);
```
because of https://github.com/indico/indico/pull/2951/files#diff-65da6c2d419c1459c8181a9673c06dc3L292 (and the extra margin/padding)